### PR TITLE
Add path input

### PR DESCRIPTION
**Pull request by iuliancristea**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Default ignored files
```
Comment (benhartley) [inline .idea/.gitignore:1]: Did these get added before the gitignore?

Comment (iuliancristea) [reply .idea/.gitignore:1]: No, the main .gitignore allows some .idea files. I can exclude all though

Comment (benhartley) [reply .idea/.gitignore:1]: Feels like they should be excluded

File: hrval-runner [hunk 2]
```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/env bash
 
-set -eu
+set -u
+
+if [ -n "${INPUT_PATH}" ]; then
+  mkdir -p "${INPUT_PATH}"
+  cd "${INPUT_PATH}"
```
Comment (benhartley) [inline hrval-runner:7]: Without `-e` if either of these fail the script will run in the wrong place

Comment (iuliancristea) [reply hrval-runner]: `mkdir -p` won't error if the path already exists;

I've added `|| exit` to `cd` which should also take care of the case when `mkdir -p` fails for e.g. permission issues.

**Review by benhartley — APPROVED** [review]

**Pull request merged**